app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -14,23 +13,32 @@
 
 public class Mood {
     private String user;
+
     private String emotionalState;
     private String description;
-    private Date date;
+    private long date; //for actually storing value in Elasticsearch, since it doesn't have a Date type
+    private static Date trueDate; //the real date as a Date object
 
     private String socialSit;
-    private Boolean isPrivate;
+    private boolean isPrivate;
```

Comment:
Boolean types are actually initialized to null, whereas boolean types are initialized to false. We want the latter here